src/components/Input/Input.stories.tsx

```diff
@@ -9,88 +13,157 @@ export default {
   component: Input,
   parameters: {
     layout: 'fullscreen',
+    componentSubtitle:
+      'Input은 사용자의 입력을 받는 컴포넌트입니다. 비제어, 제어 두 방식 모두 지원합니다.',
```

Comment:
```suggestion
      'Input은 사용자의 입력을 받는 컴포넌트입니다. 제어, 비제어 두 방식 모두 지원합니다.',
```

사소한 부분입니다.

제어, 비제어 라고 하는게 어순이 조금 더 자연스러운 것 같아요